skills/grab-cookie/scripts/credgrab.py

```
@@ -0,0 +1,635 @@
+                   dest_path: Path, prior_consumer: bytes | None) -> None:
+    """Undo a seed whose verify did not pass: restore the prior credential and
+    consumer file so a wrong or expired capture never destroys a known-good one."""
+    for store_as, prior in prior_values.items():
+        if prior is not None:
+            credstore.store(store_as, prof["name"], prior)
+        else:
+            # There was nothing before; remove what this seed added.
+            try:
+                credstore.delete(store_as, prof["name"])
+            except Exception:
+                pass
```

Ruff (0.16.1)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)